src/dams_mcda/server.R

```diff
@@ -1477,17 +1477,23 @@ server <- function(input, output, session) {
 		# download for raw values
 		output[[paste0("DownloadDam", damId, "RawTable")]] <- downloadHandler(
 		  filename = function() {
-			format(Sys.time(), paste0(dam_names[damId], "_raw_data_values_%Y-%m-%d_%H-%M-%S_%z.csv"))
+			  format(Sys.time(), paste0(dam_names[damId], "_raw_data_values_%Y-%m-%d_%H-%M-%S_%z.csv"))
 		  },
 		  content = function(file) {
-			write.csv( round(RawTable, 0), file, row.names = TRUE, quote=TRUE)
+			  write.csv( round(RawTable, 0), file, row.names = TRUE, quote=TRUE)
 		  }
 		)
-		
+
 		# download for Decision Matrix
 		output[[paste0("DownloadDecisionMatrix", damId)]] <- downloadHandler(
-		  filename = "DecisionMatrices_All.csv"	)
-		
+		  filename = function() {
+			  format(Sys.time(), "DecisionMatrices_All_%Y-%m-%d_%H-%M-%S_%z.csv")
+		  },
+		  content = function(file) {
+			  write.csv( Decisions, file, row.names = TRUE, quote=TRUE)
```

Comment:
@elbfox we might not need rownames, I will launch this branch to shiny2 for you to decide

Reply:
doesn't seem to be functional, investigating

Reply:
I did the extension as .csv instead of .xlsx...I'll go fix it in #188. Sorry!

Reply:
@elbfox There is no Decisions object anymore. Does it need to be re-added or is there a different data object you are trying to serve as a download?

Reply:
oh its a existing file; will update this branch